libs/nx-vite/src/generators/react/files/index.html__template__

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/src/favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title><%= projectName %> App</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/main.tsx"></script>
```

Comment:
I wonder if we should solve config in this PR?

Otherwise maybe we should write a blog post on how to configure config.js for NX + vite

Reply:
Yeah not stressed. I might need a quick primer on this.